client/jetpack-cloud/sections/settings/advanced-credentials/credentials-form/index.tsx

```diff
@@ -0,0 +1,245 @@
+				<FormFieldset>
+					<FormLabel htmlFor="protocol-type">{ translate( 'Credential Type' ) }</FormLabel>
+					<FormSelect
+						name="protocol"
+						id="protocol-type"
+						// value={ get( form, 'protocol', 'ssh' ) }
+						// onChange={ handleFieldChange }
+						// disabled={ formIsSubmitting }
+					>
+						<option value="ssh">{ translate( 'SSH/SFTP' ) }</option>
+						<option value="ftp">{ translate( 'FTP' ) }</option>
+					</FormSelect>
+				</FormFieldset>
+
+				<div className="credentials-form__row">
+					<FormFieldset className="credentials-form__server-address">
+						<FormLabel htmlFor="host-address">
+							{ /* { labels.host || translate( 'Server Address' ) } */ }
+							{ translate( 'Server Address' ) }
+						</FormLabel>
+						<FormTextInput
+							name="host"
+							id="host-address"
+							placeholder={ translate( 'YourGroovyDomain.com' ) }
+							// value={ get( form, 'host', '' ) }
+							// onChange={ handleFieldChange }
+							// disabled={ formIsSubmitting }
+							// isError={ !! formErrors.host }
+						/>
+						{ /* { formErrors.host && <FormInputValidation isError={ true } text={ formErrors.host } /> } */ }
+					</FormFieldset>
+
+					<FormFieldset className="credentials-form__port-number">
+						{ /* <FormLabel htmlFor="server-port">{ labels.port || translate( 'Port Number' ) }</FormLabel> */ }
+						<FormLabel htmlFor="server-port">{ translate( 'Port Number' ) }</FormLabel>
+						<FormTextInput
+							name="port"
+							id="server-port"
+							placeholder="22"
+							// value={ get( form, 'port', '' ) }
+							// onChange={ handleFieldChange }
+							// disabled={ formIsSubmitting }
+							// isError={ !! formErrors.port }
+						/>
+						{ /* { formErrors.port && <FormInputValidation isError={ true } text={ formErrors.port } /> } */ }
+					</FormFieldset>
+				</div>
+
+				<FormFieldset className="credentials-form__path">
+					<FormLabel htmlFor="wordpress-path">
+						{ /* { labels.path || translate( 'WordPress installation path' ) } */ }
+						{ translate( 'WordPress installation path' ) }
+					</FormLabel>
+					<FormTextInput
+						name="path"
+						id="wordpress-path"
+						placeholder="/public_html/wordpress-site/"
+						// value={ get( form, 'path', '' ) }
+						// onChange={ handleFieldChange }
+						// disabled={ formIsSubmitting }
+						// isError={ !! formErrors.path }
+					/>
+					{ /* { formErrors.path && <FormInputValidation isError={ true } text={ formErrors.path } /> } */ }
+				</FormFieldset>
+
+				<SegmentedControl>
+					<SegmentedControl.Item
+						selected={ mode === Mode.Password }
+						onClick={ () => setMode( Mode.Password ) }
+					>
+						{ translate( 'Use password' ) }
+					</SegmentedControl.Item>
+					<SegmentedControl.Item
+						selected={ mode === Mode.PrivateKey }
+						onClick={ () => setMode( Mode.PrivateKey ) }
+					>
+						{ translate( 'Use private key' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **79** times:
`translate( 'Private Key' )` *ES Score: 6*